src/main/java/com/iota/iri/service/dto/GetNeighborsResponse.java

```diff
@@ -12,7 +12,7 @@
      *  numberOfAllTransactions, numberOfRandomTransactionRequests, 
      *  numberOfNewTransactions, numberOfInvalidTransactions, numberOfSentTransactions
      * 
-     * @see {@link com.iota.iri.service.dto.GetNeighborsResponse.Neighbor}
+     * @see com.iota.iri.service.dto.GetNeighborsResponse.Neighbor
```

Comment:
@kwek20 
Make sure that this won't mess up your mdx generation. Basically the fix is correct. Thanks